frontend/src/metabase/static-viz/components/XYChart/Values/Values.tsx

```diff
@@ -92,6 +99,21 @@ export default function Values({
     });
   });
 
+  function fixSmallBarChartValues(
+    singleSeriesValues: Value[],
+    numberOfBarSeries: number,
+  ) {
+    const barWidth = innerBarScale?.bandwidth() ?? Infinity;
+    const isBarTooSmall = barWidth < 20;
```

Comment:
Should we name constants like these?

Reply:
Is that a question of whether this should not be named like this or in general whether we should be naming other constants like this too?

My take is that `barWidth < 20` doesn't really tell the reader much unless readers know more visualization context.

So, at glance, I thought it'd be better for readers to see that we considered 20px to be too small for a bar chart to have data point value shown.

This isn't a set rule or anything, but I do name variable at times when its implementation isn't obvious.

Reply:
Just a nit that we could define something like `const MIN_BAR_WIDTH = 20` and use that for comparisons.

Reply:
That's fair. I'll do that.